index.js

```diff
@@ -68,28 +108,14 @@ class ExecutorRouter extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
-
-        return executor.start(config);
-    }
+        const options = this.getFullConfig(config);
 
-    /**
-     * Stop a running or finished build
-     * @method _stop
-     * @param {Object} config               Configuration
-     * @param {Object} [config.annotations] Optional key/value object
-     * @param {String} config.buildId       Unique ID for a build
-     * @return {Promise}
-     */
-    _stop(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
+        // TODO: mark job as queued
 
-        return executor.stop(config);
+        // Note: arguments to enqueue are [queue name, job type, array of args]
+        return this.verifyConnection()
+            .then(() => this.queue.enqueue('queuedJobs', 'start', [options]));
```

Comment:
I don't see anything here https://github.com/screwdriver-cd/executor-queue-worker/pull/2/files which reads from `queuedJobs`, is that WIP ?